### PR TITLE
NFT Offers Endpoint

### PR DESCRIPTION
**Pull request by benisgold**
Fixes APP-####

## What changed (plus any additional context for devs)
- graphql config for nft offers endpoint
- react query hook for nft offers endpoint

### CODE REVIEW

File: src/resources/nftOffers/index.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { arcClient } from '@/graphql';
+import { GetNftOffersQuery, SortCriterion } from '@/graphql/__generated__/arc';
+import { createQueryKey } from '@/react-query';
+import { useQuery } from '@tanstack/react-query';
+
+const STALE_TIME = 60000; // 1 minute
```
Comment (benisgold) [inline src/resources/nftOffers/index.ts:6]: using a pretty low stale time bc offers are pretty spammy in general. nfts i've looked at have TONS of offers, and many of them expire quickly

**Review by estrattonbailey — APPROVED** [review]
LGTM!

File: src/resources/nftOffers/index.ts (hunk 1 above)
Comment (skylarbarrera) [reply src/resources/nftOffers/index.ts:6]: may be worth noting here or in linear that we should probably change this for prod, maybe im tripping but hitting this every min is expensive

**Review by skylarbarrera — APPROVED** [review]

File: src/graphql/config.js [hunk 2]
```diff
@@ -17,4 +17,11 @@ exports.config = {
       url: 'https://api.thegraph.com/subgraphs/name/ianlapham/uniswapv2',
     },
   },
+  arc: {
+    document: './queries/arc.graphql',
+    schema: {
+      method: 'GET',
+      url: 'https://arc-graphql.rainbow.me/graphql',
```
Comment (estrattonbailey) [inline src/graphql/config.js:24]: Can you set this up to use the staging URL when not in prod? Peep `resources/f2c` for example

File: src/resources/nftOffers/index.ts (hunk 1 above)
Comment (estrattonbailey) [reply src/resources/nftOffers/index.ts:6]: Yeah agree, if a user wants up-to-the-second resolution they should probably use a purpose-built solution. Maybe a discussion, but like every 3-5min seems better?

**Review by estrattonbailey — APPROVED** [review]
Looks sick, let's just confirm stale time and use conditional URLs

File: src/graphql/config.js (hunk 2 above)
Comment (benisgold) [reply src/graphql/config.js:24]: aye

**Pull request merged**